### ISSUE TITLE
maxBuffer exceeded

### ISSUE THREAD

**Issue by dominikwilkowski** [Manabu-GT/grunt-auto-install#12]
Hey there,

I am getting the following error:
```shell
Warning: stderr maxBuffer exceeded. Use --force to continue.
```

Can we get around this?

**Comment by dominikwilkowski** [comment]
I suggest to run `npm install -q` instead of `npm install` when stderr is `false`.

Any time for a fix?

**Comment by Manabu-GT** [comment]
probably this weekend. If you are in hurry, please send me a pull request.

Thanks.

**Comment by dominikwilkowski** [comment]
Hey that's pretty fast :)
Thanks

**Comment by Manabu-GT** [comment]
Sorry for the delay. This should be fixed in v0.2.4.

**Comment by dominikwilkowski** [comment]
:+1:
